src/ResourceManager/EventHub/Commands.EventHub/Commands.EventHub.csproj

```diff
@@ -86,16 +111,12 @@
     <None Include="..\AzureRM.EventHub.psd1">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Include="help\Test-AzureRmEventHubName.md" />
     <None Include="Microsoft.Azure.Commands.EventHub.format.ps1xml">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
     <None Include="MSSharedLibKey.snk" />
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="StartupScripts\*.ps1">
```

Comment:
Please add this back in, it will be needed in the future if any aliases are added.

Reply:
Resolved